rootfs/etc/services.d/vdlm2dec/run

```diff
@@ -7,7 +7,7 @@
 if [ -n "${ENABLE_VDLM}" ]; then
 
 	VDLM_BIN="/usr/local/bin/vdlm2dec"
-	VDLM_CMD=("-J" "-G" "-E" "-U" "-g" "$GAIN" "-i" "$STATION_ID_VDLM")
+	VDLM_CMD=("-J" "-G" "-E" "-U" "-g" "$GAIN_VDLM" "-i" "$STATION_ID_VDLM")
```

Comment:
Hi @fredclausen,

To allow backwards compatibility with `GAIN`, I suggest we do something like:

```bash
if [ -n "$GAIN" ]; then
    GAIN_VDLM="$GAIN"
fi
```

What do you think?

Reply:
Excellent call...done